datafusion/sql/tests/cases/plan_to_sql.rs

```diff
@@ -885,6 +885,7 @@ fn test_table_scan_pushdown() -> Result<()> {
     let query_from_table_scan_with_projection = LogicalPlanBuilder::from(
         table_scan(Some("t1"), &schema, Some(vec![0, 1]))?.build()?,
     )
+    .project(vec![col("id"), col("age")])?
```

Comment:
Updated test so that there is Projection that matches TableScan projection, there could not be wildcard Projection with TableScan with projection